modin/pandas/base.py

```diff
@@ -3033,6 +3100,15 @@ def to_excel(
     ):  # pragma: no cover  # noqa: PR01, RT01, D200
         """
         Write object to an Excel sheet.
+
+        Parameters Support Status:
```

Comment:
Good point! Actually lazy execution mostly relates to OmniSci (non-IO) or some advanced features with Ray/Dask engines (batch pipeline), so `Non-lazy` flag should be placed almost everywhere that will lead to tables bloating. My suggestion is to replace `Non-lazy` -> `Lazy` flag to avoid such issue. @RehanSD @YarShev @anmyachev what doy you think?

Reply:
Currently, we have `Non-lazy` flag that is only applicable to OmniSci, given the description for this flag. I think all the flags should be as generic as possible to all engines. So, yes, I think `Lazy` flag makes sense to use and we can put it not only for OmniSci related operations, but for Ray/Dask related functions (e.g., transpose).